### DISCUSSION TITLE
why default CONGESTION_CONTROL is cubic not bbr

### DISCUSSION THREAD

**Discussion by ycchildcoder** [microsoft/msquic#4005] [Q&A] [answered]
#define QUIC_CONGESTION_CONTROL_ALGORITHM_DEFAULT   QUIC_CONGESTION_CONTROL_ALGORITHM_CUBIC

**Comment by nibanks** [comment, marked as answer]
Cubic is our default congestion control because the BBR implementation is v1 of the spec (the latest draft version is v3), which has known issues in widespread deployments, and the code isn't as well tested/verified as Cubic. Use BBR at your own risk.

**Reply by ycchildcoder** [reply]
thanks
